crates/astria-cli/src/cli/sequencer.rs

```diff
@@ -31,6 +31,8 @@ pub enum Command {
     },
     /// Command for sending balance between accounts
     Transfer(TransferArgs),
+    InitBridgeAccount(InitBridgeAccountArgs),
+    BridgeLock(BridgeLockArgs),
```

Comment:
Would maybe be nice to have doc comments on these variants so the `--help` command looks more consistent.